### ISSUE TITLE
Allow to specify php data reference function FQN in twig comment

### ISSUE THREAD

**Issue by klesun** [klesun/deep-assoc-completion#119]
Considering we have such php class:
```php
namespace SomeNamespace;
class SomeClass {
    private function makeTemplateData() {
        return [
            'userName' => 'Vasya', 
            'paymentMethods' => [
                ['name' => 'Paypal', 'countries' => ['US', 'LV']],
                ['name' => 'Swedbank', 'countries' => ['ES', 'LV']],
            ],
        ];
    }
}
```

The comment format is subject to change, but for now let's go with:
```twig
{# data-source = \SomeNamespace\SomeClass::makeTemplateData() #}

<div>
    <span>Hello, {{  }}</span> <!-- should suggest: "userName", "paymentMethods" -->
</div>
```

**Comment by klesun** [comment]
Would be nice to also have it work in for loops, key access, etc...

**Comment by klesun** [comment]
![image](https://user-images.githubusercontent.com/5202330/67644126-40efaa00-f927-11e9-972f-e703c898d36d.png)
That's a start

**Comment by klesun** [comment]
Now we're talking
![image](https://user-images.githubusercontent.com/5202330/67646138-58359400-f935-11e9-854f-2ed370cb97d0.png)

**Comment by klesun** [comment]
Released in [2019.10.28.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/71246)
